nucypher/crypto/powers.py

```diff
@@ -253,7 +245,7 @@ class DecryptingPower(KeyPairBasedPower):
 
 class RitualisticPower(KeyPairBasedPower):
     _keypair_class = RitualisticKeypair
-    _default_private_key_class = ferveo_py.Keypair
+    _default_private_key_class = Keypair
```

Comment:
It's unclear which Keypair this is, even though I know it's intended to be from ferveo.  Let's use a namespace here.